news/create-newsfile.py

```diff
@@ -62,10 +62,12 @@ def create_block(block_name, files):
     block = '## {}\n\n'.format(block_name)
     for f in files:
         entry = ''
-        try:
-            pr_id = re.findall(r'\d+.md$', f.name)[0][:-3]
-        except IndexError:
+        match_pr_id = re.search(r'(\d+)(-\d+)?.md$', f.name)
```

Comment:
Please update the README, to document this possibility, thanks :)

Reply:
Done, thank you.

Reply:
Thanks!